Sources/SwiftProtobuf/FieldTag.swift

```diff
@@ -21,49 +21,45 @@
 /// for validity because they are guaranteed correct at initialization time.
 internal struct FieldTag: RawRepresentable {
 
-  typealias RawValue = UInt32
+      typealias RawValue = UInt32
 
-  /// The raw numeric value of the tag, which contains both the field number and
-  /// wire format.
-  let rawValue: UInt32
+      /// The raw numeric value of the tag, which contains both the field number and
+      /// wire format.
+      let rawValue: UInt32
 
-  /// The field number component of the tag.
-  var fieldNumber: Int {
-    return Int(rawValue >> 3)
-  }
+      /// The field number component of the tag.
+      var fieldNumber: Int { return Int(rawValue >> 3) }
 
-  /// The wire format component of the tag.
-  var wireFormat: WireFormat {
-    // This force-unwrap is safe because there are only two initialization
-    // paths: one that takes a WireFormat directly (and is guaranteed valid at
-    // compile-time), or one that takes a raw value but which only lets valid
-    // wire formats through.
-    return WireFormat(rawValue: UInt8(rawValue & 7))!
-  }
+      /// The wire format component of the tag.
+      var wireFormat: WireFormat {
+            // This force-unwrap is safe because there are only two initialization
+            // paths: one that takes a WireFormat directly (and is guaranteed valid at
+            // compile-time), or one that takes a raw value but which only lets valid
+            // wire formats through.
+            return WireFormat(rawValue: UInt8(rawValue & 7))!
+      }
 
-  /// A helper property that returns the number of bytes required to
-  /// varint-encode this tag.
-  var encodedSize: Int {
-    return Varint.encodedSize(of: rawValue)
-  }
+      /// A helper property that returns the number of bytes required to
+      /// varint-encode this tag.
+      var encodedSize: Int { return Varint.encodedSize(of: rawValue) }
 
-  /// Creates a new tag from its raw numeric representation.
-  ///
-  /// Note that if the raw value given here is not a valid tag (for example, it
-  /// has an invalid wire format), this initializer will fail.
-  init?(rawValue: UInt32) {
-    // Verify that the field number and wire format are valid and fail if they
-    // are not.
-    guard rawValue & ~0x07 != 0,
-      let _ = WireFormat(rawValue: UInt8(rawValue % 8)) else {
-      return nil
-    }
-    self.rawValue = rawValue
-  }
+      /// Creates a new tag from its raw numeric representation.
+      ///
+      /// Note that if the raw value given here is not a valid tag (for example, it
+      /// has an invalid wire format), this initializer will fail.
+      init?(rawValue: UInt32) {
+            // Verify that the field number and wire format are valid and fail if they
+            // are not.
+            guard rawValue & ~0x07 != 0,
+                  let _ = WireFormat(rawValue: UInt8(rawValue % 8)) else {
+                  return nil
```

Comment:
The continuation of the` guard` the the body of the `else` having the same indent doesn't seem so great.

Reply:
Yeah, the `else {` should have moved down to the next line at the original indentation. I have specific test cases for situations similar to this, so I need to track down why it's not happening here.